### PR TITLE
Add organization model

### LINKED ISSUE #14: An admin should be able to create an new organization

### PR DESCRIPTION
**Pull request by grimm**
Fixes #14

### CODE REVIEW

**Comment by grimm** [comment]
This is ready for review.  I did a temp fix for the new manager layout.  I wanted to do separate branches for moving the maps, layers, and the multimaps views over to the new layout.

File: app/models/organization.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class Organization < ActiveRecord::Base
+  has_many :organization_users
+  has_many :users, through: :organization_users
+
+  has_many :maps
+  has_many :multimaps
```
Comment (teknofire) [inline app/models/organization.rb]: Missing `has_many :geojson_layers ` and `has_many :wms_layers`

File: app/models/organization.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+class Organization < ActiveRecord::Base
```
Comment (teknofire) [inline app/models/organization.rb]: Make sure to use the `friendly_id` stuff for this based on the name of the organization

Make sure organization names are required and uniq as well

File: app/controllers/organizations_controller.rb [hunk 3]
```diff
@@ -0,0 +1,62 @@
+class OrganizationsController < ApplicationController
+  include GinaAuthentication::Users
```
Comment (teknofire) [inline app/controllers/organizations_controller.rb]: This include is not needed for this controller

**Comment by grimm** [comment]
Add organizations to header menu, make sure to test for auth.

**Pull request merged**